src/applications/check-in/utils/navigation/navigation.unit.spec.js

```diff
@@ -108,8 +108,8 @@ describe('Global check in', () => {
         );
         expect(form.find(page => page === URLS.DEMOGRAPHICS)).to.exist;
       });
-      it('should skip a page that has been updated more than 72 hours ago but within three calendar days', () => {
-        MockDate.set('2022-01-04T14:00:00.000-05:00');
+      it('should skip a page that has been updated more than 168 hours ago but within seven calendar days', () => {
```

Comment:
Hmm, I don't think the test matches its description? I'm getting a 110hour difference.

Reply:
I was looking at `demographicsConfirmedAt` instead of `emergencyContactConfirmedAt`!